connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -205,6 +205,9 @@ private void onCommitCompleted(Throwable error, long seqno) {
             } else {
                 log.debug("{} Finished offset commit successfully in {} ms",
                         this, time.milliseconds() - commitStarted);
+                if (committedOffsets != null) {
```

Comment:
It would be nice to add a short comment about the case when `committedOffsets` is null. I would have expected that only one of `error` and `committedOffsets` would be null, but I see there is a case below where they both are.